back-office/public/index.html

```diff
@@ -7,6 +7,8 @@
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="http://diegoddox.github.io/react-redux-toastr/7.0/react-redux-toastr.min.css" rel="stylesheet" type="text/css">
```

Comment:
You should add this css in App.jsx file, in that way : 
`import 'react-redux-toastr/lib/css/react-redux-toastr.min.css'`

Because having a `<link href..>` pointing on github doesn't give any garantee to be always available.